utils/paddedToAddress.ts

```diff
@@ -0,0 +1,17 @@
+/**
+ * Remove padding from address
+ *
+ * @param address
+ * @returns
+ */
+export const paddedToAddress = (address: string) => {
+  if (address.slice(0, 2) === '0x') {
```

Comment:
maybe

```suggestion
  if (address.startsWith('0x')) {
```

not a big deal but i think it reads easier